projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -405,6 +383,31 @@ async def _clean_room(self) -> None:
                 self._emit(LogLevel.INFO, "clean", "Loader deleted.")
             del self._room_locks[self._room_id]
 
+    def _on_global_awareness_event(self, type: str, changes: tuple[dict[str, Any], Any]) -> None:
```

Comment:
`type` is a keyword in Python, better to use something else.
```suggestion
    def _on_global_awareness_event(self, topic: str, changes: tuple[dict[str, Any], Any]) -> None:
```